### DISCUSSION TITLE
checkout not working in woocommerce checkout

### DISCUSSION THREAD

**Discussion by ashishkerni** [woocommerce/woocommerce#47162] [Checkout Flow]
checkout fields can not be changed eg. i want to set phone number required in shipping ... but even after select the required checkbox .. phone number field shown as optional..  please set this asap.. for shipping products phone no is compulsory to contact the customer .

**Comment by lanej0** [comment]
Hi @ashishkerni -- we haven't heard about any widespread problems on this front.  My bet would be that there's a plugin or theme conflict going on here that's giving you trouble.  My first suggestion would be to try a default theme and to try deactivating any plugins that may be affecting your cart/checkout flows.

If you need additional help, your best bet might be to [open an issue](https://github.com/woocommerce/woocommerce/issues) or to contact WooCommerce support (if we're dealing with a paid plugin/subscription)

**Comment by senadir** [comment]
I have seen a couple of reports for this and it's always been a plugin that's editing the Checkout div, can you disable plugins and see what's causing this?
